generated/nidaqmx/constants.py

```diff
@@ -234,9 +238,9 @@ class DataTransferActiveTransferMode(Enum):
 
 
 class DeassertCondition(Enum):
-    ON_BOARD_MEMORY_MORE_THAN_HALF_FULL = 10237  #: Deassert the signal when more than half of the onboard memory of the device fills.
-    ON_BOARD_MEMORY_FULL = 10236  #: Deassert the signal when the onboard memory fills.
-    ONBOARD_MEMORY_CUSTOM_THRESHOLD = 12577  #: Deassert the signal when the amount of space available in the onboard memory is below the value specified with **rdy_for_xfer_event_deassert_cond_custom_threshold**.
+    ONBRD_MEM_MORE_THAN_HALF_FULL = 10237  #: Deassert the signal when more than half of the onboard memory of the device fills.
+    ONBRD_MEM_FULL = 10236  #: Deassert the signal when the onboard memory fills.
+    ONBRD_MEM_CUSTOM_THRESHOLD = 12577  #: Deassert the signal when the amount of space available in the onboard memory is below the value specified with **rdy_for_xfer_event_deassert_cond_custom_threshold**.
```

Comment:
This one is a mixed bag. The old names were inconsistent (ON_BOARD vs ONBOARD), so fixing the inconsistency is great. However, I don't like ONBRD, and would prefer ONBOARD

Reply:
Updated enum values as in expected constants.py